GreenhouseNode/GreenhouseNode/temp/LoRaWAN_Task.c

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
What's the point of the temp folder? Is it for temporary files or for temperature stuff, I'm guessing it's temporary files for the POC?